src/bindgen/parser.rs

```diff
@@ -626,33 +626,40 @@ impl Parse {
             return;
         }
 
+        let loggable_item_name = {
```

Comment:
This is a bit unfortunate in the sense that it runs this code unconditionally... But can improve in a follow-up.